<?xml version="1.0"?>
<launch>
    <arg name="rvizconfig" default="$(find ompl_example_2d)/rviz/ompl.rviz" />
  
    <!-- map_server -->
    <!-- <node pkg="map_server" name="map_server" type="map_server" output="screen" args="$(find ompl_example_2d)/maps/map_home.yaml"/> -->
    
    <!-- RViz -->
    <!-- <node pkg="rviz" name="rviz" type="rviz" output="screen" args="-d $(arg rvizconfig)"/> -->
    
    <!-- ompl-based motion planning node -->
    <node pkg="ompl_example_2d" name="ompl_example_2d_node" type="ompl_example_2d_node" output="screen" />
    
</launch>
